yolk/parser.py

```diff
@@ -1,36 +1,74 @@
 import argparse
 
-def parse_args(args):
-    """ Parse the arguments.
+def check_args(parsed_args):
```

Comment:
need path

Reply:
If user selects training mode, dataset and data path should be provided. It is checked in check_args() function.